src/index.js

```diff
@@ -40,23 +39,34 @@ const customerDashboard = document.querySelector(".customer");
 const landingPage = document.querySelector(".welcome");
 const managerSearchResults = document.querySelector(".manager-search");
 const customerMakeBooking = document.querySelector(".customer-booking");
+const availableRoomsDisplay = document.querySelector(".display-all-rooms");
 const totalRoomsAvailableToday = document.querySelector(
-  ".number-rooms-available"
-);
+  ".number-rooms-available");
 const percentRoomsOccupiedToday = document.querySelector(".percent-occupied");
 const totalRevenueToday = document.querySelector(".total-revenue");
 const customerPastBookings = document.querySelector(".dashboard-past-bookings");
 const customerTodayBooking = document.querySelector('.dashboard-today-booking');
 const customerFutureBooking = document.querySelector('.dashboard-future-bookings');
 const customerPastSpending = document.querySelector('.customer-amount-spent');
+const customerInputDate = document.querySelector('.booking-selection');
+const inputDateSubmitButton = document.querySelector('.submit-date');
+const addAvailableRoomToDisplay = document.querySelector('.add-available-rooms');
+
+
+let userName;
+let password;
+let bookingDate;
 
-submitBtn.addEventListener("click", createUser);
-userNameInput.addEventListener("input", collectUserName);
```

Comment:
It might be less memory-draining to assign these on button press rather than having an event listener. But it works this way too!